middleware/src/middleware.go

```diff
@@ -748,7 +748,24 @@ func (middleware *Middleware) EnableRootLogin(toggleAction rpcmessages.ToggleSet
 	return rpcmessages.ErrorResponse{Success: true}
 }
 
-// SetRootPassword sets the systems root password
+// EnablePasswordLogin enables/disables the ssh login with a password (in addition to ssh keys)
+// and returns a ErrorResponse indicating if the call was successful.
+func (middleware *Middleware) EnablePasswordLogin(toggleAction rpcmessages.ToggleSettingArgs) rpcmessages.ErrorResponse {
+	log.Printf("Executing 'Enable password login: %t' via the config script.\n", toggleAction)
+	out, err := middleware.runBBBConfigScript([]string{determineEnableValue(toggleAction), "pwlogin"})
+	if err != nil {
+		errorCode := handleBBBScriptErrorCode(out, err, nil)
+		return rpcmessages.ErrorResponse{
+			Success: false,
+			Message: strings.Join(out, "\n"),
+			Code:    errorCode,
+		}
+	}
+
+	return rpcmessages.ErrorResponse{Success: true}
+}
+
+// SetRootPassword sets the system main ssh/login password
 func (middleware *Middleware) SetRootPassword(args rpcmessages.SetRootPasswordArgs) rpcmessages.ErrorResponse {
```

Comment:
Should probably be renamed as well, or?

Reply:
You mean the `SetRootPassword`? If possible, yes, but I did not want to introduce any breaking changes.

Reply:
Yes `SetRootPassword`. Would need a rename in the App as well. But is needed anyway to keep things clean.

Reply:
Ok, let's do it now. Better than after release.